index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css" />
+    <title>A Tribute Page</title>
+</head>
+<body>
+    <!--
+       <div id="main">
+        <div id="title"><h1>Dr Norman Borlaug</h1></div>
+        <Figure id="img-div">
+            <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="image">
+            <div id="img-caption"><p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p></div>
+        </Figure>
+        <div id="tribute-info">
+          <a href="#" id="tribute-link" target="_blank">The man who saved a billion lives</a>
+        </div>
+    </div>
+    -->
+    <main id="main" style="color: #333">
```

Comment:
In line CSS should not be used, kindly follow the best CSS practices by going through this [link ](https://medium.com/before-semicolon/50-css-best-practices-guidelines-to-write-better-css-c60807e9eee2)